main.scss

```diff
@@ -1,4 +1,7 @@
-@import "o-colors/main";
+@import
```

Comment:
I didn't know you could do this, but we do one import statement per line just to be consistent with other components